### PR TITLE
add pet-app stg service for testing

### PR DESCRIPTION
**Pull request by tpetkovic**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by naikajah** [comment]
The only question I have is what are we trying to solve or prove by applying this test app to aat front door?

1. Shouldn't this be done to sandbox environment if we want to test anything?
2. What testing are we doing with this dummy application that we haven't done with the divorce" application?

File: environments/stg/global.tfvars [hunk 1]
```diff
@@ -49,4 +49,14 @@ frontends = [
       SQLI = ["942440"]
     }
   },
+  {
+    name             = "pet-app1"
```
Comment (timja) [inline environments/stg/global.tfvars]: shouldn't this be an actual app name? surely there's not one called pet-app1?

File: environments/stg/global.tfvars [hunk 2]
```diff
@@ -49,4 +49,14 @@ frontends = [
...
+    custom_domain    = "aat.platform.hmcts.net"
+    backend_domain   = ["firewall-prod-int-palo-aat.uksouth.cloudapp.azure.com"]
+    certificate_name = "wildcard-aat-platform-hmcts-net"
+    disabled_rules = {
```
Comment (timja) [inline environments/stg/global.tfvars]: do these rules need to be disabled?

File: environments/stg/global.tfvars (hunk 1 above)
Comment (tpetkovic) [reply environments/stg/global.tfvars]: That's the name of the app that has been spun up temporarily for testing purposes

Comment (timja) [reply environments/stg/global.tfvars]: what are you testing?

File: environments/stg/global.tfvars (hunk 2 above)
Comment (tpetkovic) [reply environments/stg/global.tfvars]: I will remove them, this is being done temporarily for testing purposes

File: environments/stg/global.tfvars (hunk 1 above)
Comment (tpetkovic) [reply environments/stg/global.tfvars]: The different data flow to PET compared to the divorce app

File: environments/stg/global.tfvars [hunk 3]
```diff
@@ -49,4 +49,12 @@ frontends = [
       SQLI = ["942440"]
     }
   },
+  {
+    name             = "pet-app1"
+    custom_domain    = "aat.platform.hmcts.net"
```
Comment (timja) [inline environments/stg/global.tfvars]: domain verification is missing:
```
$ dig afdverify.pet-app1.aat.platform.hmcts.net

; <<>> DiG 9.10.6 <<>> afdverify.pet-app1.aat.platform.hmcts.net
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 46155
```

Comment (naikajah) [reply environments/stg/global.tfvars]: it seems it is done now

Comment (tpetkovic) [reply environments/stg/global.tfvars]: its been added now with afdverify

**Pull request merged**